libs/net/traffic_generator.py

```
@@ -164,6 +178,49 @@ def _is_process_running(vm: BaseVirtualMachine, cmd: str) -> bool:
         return False
 
 
+def _client_log_path(server_ip: str, server_port: int) -> str:
+    """Return the VM path capturing the iperf3 client output for a given server endpoint."""
+    sanitized_server_ip = server_ip.replace(":", "_").replace(".", "_")
+    return f"/tmp/{_IPERF_BIN}_client_{sanitized_server_ip}_{server_port}.log"
```

Ruff (0.16.3)

[error] Probable insecure usage of temporary file or directory: "/tmp/"

(S108)